ledger/src/blockstore.rs

```diff
@@ -2348,14 +2348,14 @@ impl Blockstore {
             "blockstore-rpc-api",
             ("method", "get_rooted_transaction_status", String)
         );
-        self.get_transaction_status(signature, &[])
+        self.get_transaction_status(signature, &HashSet::default())
     }
 
     /// Returns a transaction status
     pub fn get_transaction_status(
```

Comment:
I was concerned about this change to the signature of a public function, but it seems like nothing is calling this, in the monorepo at least.

Reply:
Yeah, I had a similar thought but monorepo (namely `solana_rpc`) built so went forward with it.